docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -1231,22 +1246,22 @@ quarkus tls lets-encrypt renew-certificate \
 During this command, TLS registry CLI reads a Let's Encrypt account information recorded during the <<lets-encrypt-issue-certificate>> step, issues a Let's Encrypt certificate request, and communicates with a Quarkus application to have ACME challenges resolved.
 
 Once the Let's Encrypt certificate chain and private key have been successfully renewed, they are converted to PEM format and copied to your application's `.letsencrypt` folder.
-TLS registry is informed that a new certificate and private key are ready and it reloads them automatically.
+The TLS registry is informed that a new certificate and private key are ready and that they are reloaded automatically.
 
 [[lets-encrypt-ngrok]]
 === Use NGrok for testing
 
 https://ngrok.com/[Ngrok] can be used to provide a secure HTTPS tunnel to your application running on localhost, and make it easy to test HTTPS based applications.
 
-Using Ngrok provides an easiest option to get started with the Quarkus Let's Encrypt ACME feature.
+Using Ngrok provides the easiest option to get started with the Quarkus Let's Encrypt ACME feature.
```

Comment:
Question/suggestion: Can we say something is the "easiest" option as it rather subjective? Could we be more direct and say "Using ngrok, you can get started with the Quarkus Let's Encrypt ACME feature."  or "ngrok provides a simplified way of getting started with the Quarkus Let's Encrypt ACME feature."